Arduino/SensorLogic/SensorLogic.ino

```diff
@@ -50,11 +53,9 @@ void loop()
   Serial.println("1 for RFID");
   Serial.println("2 for Fingerprint");
 
-  while(Serial.available()<1){
-
-  }
+  
```

Comment:
This loop is removed to allow an automatic choosing of the RFID without serial input.